CHANGELOG.asciidoc

```diff
@@ -202,6 +202,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha1...v7.0.0-alpha2[Check the
 - Rename many `icinga.*` fields to map to ECS. {pull}9294[9294]
 - Rename many `postgresql.log.*` fields to map to ECS. {pull}9303[9303]
 - Rename many `kafka.log.*` fields to map to ECS. {pull}9297[9297]
+- Add `convert_timezone` option to Elasticsearch module to convert dates to UTC. {issue}9756[9756] {pull}9761[9761]
```

Comment:
I think this changelog is in the wrong place as alpha2 is already shipped. Best directly add it to `.next` changelog.

Reply:
Moved in d9f5f1f.